python/cudf/cudf/core/column/column.py

```diff
@@ -2200,6 +2194,17 @@ def full(size: int, fill_value: ScalarLike, dtype: Dtype = None) -> ColumnBase:
     return ColumnBase.from_scalar(cudf.Scalar(fill_value, dtype), size)
 
 
+def _cudf_dtype_from_arrow_type(arrow_type: Dtype) -> Dtype:
+    if pa.types.is_decimal(arrow_type):
+        return Decimal64Dtype.from_arrow(arrow_type)
+    elif pa.types.is_struct(arrow_type):
```

Comment:
This function feels very boilerplatey - is there a better way to map cuDF dtypes to pyarrow types?

Reply:
https://github.com/rapidsai/cudf/blob/branch-21.08/python/cudf/cudf/utils/dtypes.py#L335-L344